### PR TITLE
Update pigeon documentation

### PR DESCRIPTION
**Pull request by jmattick**
Includes more information about pigeon output.

### CODE REVIEW

File: docs/classification/pigeon-annotation.md [hunk 1]
```diff
@@ -118,3 +118,54 @@ SIRV1	LexogenSIRVData	exon	7553	7814	.	-	0	gene_name "SIRV1"; gene_id "SIRV1"; t
...
+Example of gff after python modifications:
+```
+1	cshl_gene	mRNA	2600	3653	.	+	.	ID=SbiRTX436.01G000100.1;Parent=SbiRTX436.01G000100;Name=SbiRTX436.01G000100.1;biotype=protein_coding
+1	cshl_gene	five_prime_UTR	2600	2920	.	+	.	Parent=SbiRTX436.01G000100.1;Name=5UTR.1
+1	cshl_gene	exon	2600	3653	.	+	.	Parent=SbiRTX436.01G000100.1;Name=exon.1
+1	cshl_gene	CDS	2921	3586	.	+	.	Parent=SbiRTX436.01G000100.1;Name=CDS.1
+1	cshl_gene	three_prime_UTR	3587	3653	.	+	.	Parent=SbiRTX436.01G000100.1;Name=3UTR.1
+```
+
+Next, normalize using gffread.
```
Comment (velociroger-pb) [inline docs/classification/pigeon-annotation.md]: Is `normalize` the right word here? Looks like this is converting gff to gtf?

Comment (jmattick) [reply docs/classification/pigeon-annotation.md]: It's converting and also removing excess features. I can change this to "convert to a simplified gtf". 

```
 By default (i.e. without -O) only transcripts are processed, discarding any
 other non-transcript features. Default output is a simplified GFF3 with only
 the basic attributes.
```

File: docs/classification/pigeon-annotation.md [hunk 2]
```diff
@@ -118,3 +118,54 @@ SIRV1	LexogenSIRVData	exon	7553	7814	.	-	0	gene_name "SIRV1"; gene_id "SIRV1"; t
...
+Update fields to adhere to above guidelines using python.
+```
+cat sorghum.gff | python -c '
+import sys
+for line in sys.stdin:
+  fields = line.split("\t")
+  if len(fields) > 8:
+    if fields[2] == "gene":
+      fields[8] = fields[8].replace("Name=", "gene_name=")
+    if fields[0].startswith("Chr0"):
```
Comment (derekwbarnett) [inline docs/classification/pigeon-annotation.md]: Do we need to strip the reference names?

Comment (jmattick) [reply docs/classification/pigeon-annotation.md]: I believe this full example had a mix of "Chr01" and "Chr1", but for the purposes of this example we can remove this.

File: docs/classification/pigeon-annotation.md [hunk 3]
```diff
@@ -118,3 +118,50 @@ SIRV1	LexogenSIRVData	exon	7553	7814	.	-	0	gene_name "SIRV1"; gene_id "SIRV1"; t
...
+```
+
+Update fields to adhere to above guidelines using python.
+```
+cat sorghum.gff | python -c '
+import sys
+for line in sys.stdin:
+  fields = line.split("\t")
+  if len(fields) > 8:
+    if fields[2] == "gene":
```
Comment (derekwbarnett) [inline docs/classification/pigeon-annotation.md:142]: Can you add a "gene" entry to the example, to show the "Name=" label changing? It doesn't show that at the moment.

**Review by velociroger-pb — APPROVED** [review]

**Review by derekwbarnett — APPROVED** [review]

**Pull request merged**